iot-hub/Quickstarts/device-streams-proxy/device/Program.cs

```diff
@@ -32,7 +32,7 @@ public static class Program
         private static string s_port = Environment.GetEnvironmentVariable("REMOTE_PORT");
 
         // Select one of the following transports used by DeviceClient to connect to IoT Hub.
-        private static TransportType s_transportType = TransportType.Amqp;
+        private static readonly TransportType s_transportType = TransportType.Amqp;
         //private static TransportType s_transportType = TransportType.Mqtt;
```

Comment:
update the rest to be readonly

Reply:
we can't. Main() overrides them if they have been provided by the command line arguments.

Reply:
ah, you are talking about the commented code, I thought you meant the other static fields.
Fixed.